Src/BootCamp.Chapter/PersonAndBalance.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace BootCamp.Chapter
+{
+    public class PersonAndBalance
+    {
+        private string[] _personAndBalanceArray;
+        private float[] _balance;
+
+        public PersonAndBalance(string personAndBalance)
+        {
+            _personAndBalanceArray = ArrayHandler.ConvertToArray(personAndBalance);
+            _balance = ArrayHandler.ConvertToBalanceArray(personAndBalance);
+        }
+
+        public string GetName()
+        {
+            return _personAndBalanceArray[0];
+        }
+
+        public float[] GetBalanceHistory()
+        {
+            return _balance;
+        }
+
+        public float GetCurrentBalance()
+        {
+            return _balance[^1];
+        }
+
+        public float GetHighestBalance()
+        {
+            var highestBalance = _balance[0];
+            for (int i = 0; i < _balance.Length; i++)
+            {
+                if (highestBalance < _balance[i])
+                {
+                    highestBalance = _balance[i];
+                }
+            }
+            return highestBalance;
+        }
+
+        public float GetLoss()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetLoss' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5070129)